web/client/src/library/pages/docs/SourceList.tsx

```diff
@@ -24,12 +24,18 @@ export default function SourceList({
         <Input
           className="w-full !m-0"
           size={EnumSize.sm}
-          value={filter}
-          placeholder="Filter models"
-          onInput={e => {
-            setFilter(e.target.value)
-          }}
-        />
+        >
+          {({ className }) => (
+            <Input.Textfield
+              className={clsx(className, 'w-full')}
+              value={filter}
+              placeholder="Filter models"
+              onInput={e => {
```

Comment:
Missing type? Also, when and why do we need `e.stopPropagation()`?

Reply:
it stops event from being despatch up the DOM. So if you press `ArrowDown` on a page it should scroll page a bit , but in some cases you may not want that default behavior , like when you trying to open Menu Items list by pressing `ArrowDown`. In such as cases (or any cases we know that event listener is final destination) we can `stopPropagation` to not trigger accidentally handler on other DOM element